src/app/public/modules/auth-http-client/local-endpoint-checker.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  HttpClient,
+  HttpHandler
+} from '@angular/common/http';
+
+import {
+  BBAuthClientFactory
+} from '@skyux/auth-client-factory';
+
+export class LocalEndpointChecker {
+
+  public static getUrlForLocalServe(requestUrl: string,
+                                    handler: HttpHandler,
+                                    baseLocalServer: string,
+                                    fallBack: () => Promise<string>): Promise<string> {
```

Comment:
i like the use of a function for the fallback. 👏

Reply:
❤️